ddtrace/contrib/django/patch.py

```diff
@@ -0,0 +1,22 @@
+import wrapt
+
+import django
+
+def patch():
+    """Patch the instrumented methods
+    """
+    if getattr(django, '_datadog_patch', False):
+        return
+    setattr(django, '_datadog_patch', True)
+
+    _w = wrapt.wrap_function_wrapper
+    _w('django', 'setup', traced_setup)
+
+def traced_setup(wrapped, instance, args, kwargs):
```

Comment:
what happen if I call the the `django.setup()` twice? if I remember well, it's legit to manually call `django.setup()` for any kind of reason, and it means that this `traced_setup` is called twice. I think we're going to append the tracer multiple times. Can we have a test for that?